doc/api_conf.rst

```diff
@@ -44,6 +44,11 @@ Configurable settings of the :class:`dnf.Base` object are stored into a :class:`
 
     Debug messages output level, in the range 0 to 10. Default is 2.
 
+
+  .. attribute::defaultyes
+
+    Like ``assumeyes`` but will still prompt for confirmation. The default is ``False``.
```

Comment:
actually does it makes sense to set it from Python API? AFAIR Python API calls are not interactive. I woudl drop this then.